Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -343,6 +343,24 @@ public class Event: NosManagedObject, VerifiableEvent {
         return NSPredicate(format: queryString, argumentArray: arguments)
     }
     
+    @nonobjc public class func homeFeedPredicate(
```

Comment:
This is pretty much duplicates the function above. Can you refactor so that they share most of this code? I'm imagining a shared function that does all of this except for the `createdAt` condition, which this function would then add by constructing a compound predicate.